### DISCUSSION TITLE
Add 'Fn-Lock' Indicator to Software Logo

### DISCUSSION THREAD

**Discussion by Biziame** [seerge/g-helper#582] [Ideas]
I have a quick request regarding your software. Could you please add a small symbol to the software logo that indicates the status of the "fn-lock" function?
It would be really helpful for users like me who frequently use the "fn-lock" feature. Having a visual indicator in the logo would allow us to easily know whether the "fn-lock" is active or not while using the software.

**Comment by seerge** [comment]
@Biziame there is already an OSD notification when you turn FN-Lock ON/OFF (+ it's reflected with corresponding checkbox under "Extra")

**Reply by Biziame** [reply]
@seerge Visual indicator for FN-Lock status would be helpful to prevent accidental actions and facilitate immediate use of shortcuts, especially when distracted.
